src/controllers/launch/sim.launch.py

```diff
@@ -8,22 +8,6 @@
 from launch.substitutions import TextSubstitution
 import yaml
 
-env_launch = {
-    'CarGoal': 'cargoal',
-    'CarWall': 'carwall',
```

Comment:
Is this dictionary used at all in this launch file?

Reply:
Yep was in line 61 (before other changes) as 

`os.path.join(pkg_environments, f'{env_launch[env]}.launch.py')),`

I think the main/only purpose was to convert to lowercase for the filename, so I just replaced it with env.lower()